src/components/Common/SearchByMultipleFields.tsx

```diff
@@ -110,6 +114,14 @@ const SearchByMultipleFields: React.FC<SearchByMultipleFieldsProps> = ({
         e.preventDefault();
         setOpen(true);
       }
+      if ((e.ctrlKey || e.metaKey) && e.key === "k") {
```

Comment:
LGTM so far, but check the expected behavior checklist in the issue. There's a few items missing (displaying the shortcut hint).